admin/release.py

```diff
@@ -658,11 +650,6 @@ def upload_python_packages(scratch_directory, target_bucket, top_level,
     :param bytes target_bucket: S3 bucket to upload packages to.
     :param FilePath top_level: The top-level of the flocker repository.
     """
-    if setuptools_version != '3.6':
```

Comment:
:question: Is it not useful to have the equivalent of this, but for the new necessary setuptools version?